crates/driver/example.toml

```diff
@@ -1,3 +1,6 @@
+## Currently only required for CoW AMMs
+#db-url = "postgres://"
```

Comment:
Sorry that I didn't think about this earlier but could we avoid adding a requirement on the DB in the driver case? For example we could have the autopilot actually index the cow amms and the driver could just keep the `surplus_capturing_jit_order_owners` from the auctions in-memory.
WDYT?